src/common/commonutils/FileUtils.c

```diff
@@ -998,13 +1000,15 @@ int ReplaceMarkedLinesInFile(const char* fileName, const char* marker, const cha
                 }

```

Comment:
In the original fix proposal, the flose(tempHandlee) was removed. This line introduces doble close, because the file descriptor is shared by the file stream pointer returned by the fdopen(tempDescriptor) call.

Reply:
Taking a look now, fix will be in a new PR.

Reply:
[https://github.com/Azure/azure-osconfig/pull/773](https://github.com/Azure/azure-osconfig/pull/773)